2018-03-27T14:19:36.633Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-894b1ea3]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-27T14:19:36.635Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-894b1ea3]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-894b1ea3]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer\n\n    at _initializeChannel.then.then.then.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:927:34)\n    at <anonymous>"}$
2018-03-27T14:20:03.804Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-745b18f6]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-27T14:20:03.805Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-745b18f6]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: error executing chaincode: transaction returned with failure: [composerchannel-745b18f6]Calling chaincode Invoke() returned error response [Error: Failed to add object with ID 'undefined' in collection with ID 'Asset:org.gov.fundtracker.MMDA' as the object already exists]. Sending ERROR message back to peer\n\n    at _initializeChannel.then.then.then.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:927:34)\n    at <anonymous>"}$
